javascript/javascript2/week2/movies/movies.js

```diff
@@ -0,0 +1,1729 @@
+const movies = [
```

Comment:
this is not okay...you should not make your own movies variable. Instead include javascript2/week2/homework/movies.js into your .html file